src/websockets/WebSocketProvider/index.tsx

```diff
@@ -82,6 +87,12 @@ const WebSocketProvider = ({ children }: PropsWithChildren) => {
       setLoading(false);
     });
 
+    socket.on('quitting', (message) => {
+      const playerQuitting: Player = JSON.parse(message);
+      playerQuitting.quitting = true;
+      setQuitting(true);
+    });
+
```

Comment:
We can safely remove this handler because on leave the server doesn't responds back